Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/IndirectSampleChangerTest.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import (absolute_import, division, print_function)
+
+import unittest
+from mantid.simpleapi import IndirectSampleChanger
+from mantid.api import mtd
+
+
+class IndirectSampleChangerTest(unittest.TestCase):
+    def test_sampleChanger(self):
+        """
+        Basic test for sample changer
+        """
+        IndirectSampleChanger(FirstRun=72462, LastRun=72465, NumberSamples=1, Instrument='IRIS', Analyser='graphite',
+                              Reflection='002', SpectraRange='3, 50', ElasticRange='-0.5, 0',
+                              InelasticRange='0, 0.5', GroupingMethod='All')
+        scan_ws = mtd['iris72462_to_72465_s0_scan_eisf']
+        self.assertEqual(round(scan_ws.readY(0)[0], 7), 1.0519965)
+        self.assertEqual(round(scan_ws.readY(1)[0], 7), 1.0452707)
+
+    def test_multiple_samples(self):
```

Comment:
We've got some good tests here. It's good practice to also add tests that verify failure behaviour.

Can we add some tests that test the invalid scenarios being checked by `validateInputs`?